example/src/components/ChatWindow.tsx

```diff
@@ -247,11 +241,12 @@ export const ChatWindow = ({localPeer}: {localPeer?: HMSLocalPeer}) => {
             />
           </View>
         )}
-        <FlatList
-          ref={scollviewRef}
+        <FlashList
```

Comment:
[Flashlist docs](https://shopify.github.io/flash-list/docs/usage) suggests to remove `key` prop in `renderItem` hierarchy.

Remove `key` prop at line 265